docker/docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+version: '3'
+services:
+
+  damian:
+    image: aleph-node:latest
+    container_name: damian
+    entrypoint: ["/bin/sh","-c"]
+    command:
+      - |
+         ./aleph-node purge-chain --chain testnet1 --base-path /tmp/Damian -y
+         ./aleph-node --validator --chain testnet1 --base-path /tmp/Damian --name Damian --node-key-file /tmp/Damian/libp2p_secret --rpc-port 9933 --ws-port 9944 --port 30334 --rpc-cors all --rpc-methods Safe --execution Native --no-prometheus --no-telemetry --reserved-only --reserved-nodes /ip4/172.28.1.1/tcp/30334/p2p/12D3KooWAbWDCsZiA6EXrXh33E7hBYUFK53b5CSajoPsYL5n2Zdm /ip4/172.28.1.2/tcp/30334/p2p/12D3KooWQdQW1Laeq869b4pdDGSVZebbjY9h9wmdNnauFeFWUKup /ip4/172.28.1.3/tcp/30334/p2p/12D3KooWD4UoT2euDnFwnSRhfUjMt6DWkMTGWV8hQm2tuNgX4j3V /ip4/172.28.1.4/tcp/30334/p2p/12D3KooWQzGBGxYx3Su9EaFMS1qtVHc2zSYdTmgRTeRGjGXj9uNg --session-period 25
```

Comment:
I propose to set miliseconds_per_block to 1000.

Reply:
Done